crates/typst/src/math/mod.rs

```diff
@@ -158,37 +158,60 @@ pub static MATH: Category;
 pub fn module() -> Module {
     let mut math = Scope::deduplicating();
     math.category(MATH);
-    math.define_elem::<EquationElem>();
```

Comment:
I'd prefer to keep the elements listed as before. The file comments just add duplication with the real file system layout (that could get out of date) and also introduce inconsistency with the other categories (non-math).